Feature/m0137 w 732 download install sdk refactor download links (#462)
MicroEJ/docs: SDKUserGuide/installSDKDistributionOther.rst

Install Other SDK Distributions
===============================

.. toctree::
   :hidden: 

   installSDKDistributionPortable
   installSDKDistribution21_11

> This section applies when the :ref:`latest SDK Distribution <sdk_installation_latest>` cannot be installed:

Reviewer: ```suggestion
This section applies when the installation of the :ref:`latest SDK Distribution <sdk_installation_latest>` via the installer does not fit your case:
```

> - you want to install the latest SDK Distribution based on JDK8, see :ref:`sdk_installation_21_11` section.

Reviewer: ```suggestion
- you want to install the latest SDK Distribution compatible with JDK8, see :ref:`sdk_installation_21_11` section.
```

Install Other SDK Distributions
===============================

.. toctree:: 

  installSDKDistributionPortable

>   installSDKDistribution21_11

Author: I rewrote the introduction, please have a look

> The following table gathers all SDK Distributions links for SDK version ``5.x``.

Author: done

.. list-table::
   :widths: 10 10 20 20 20 20 15 15

   * - **SDK Distribution**
     - **JDK Version**
     - **Windows**
     - **Linux**
     - **macOS x86_64 (Intel chip)**
     - **macOS aarch64 (M1 chip)**
     - **SDK Version**
     - **Eclipse Version**
   * - ``22.06``
     - ``11``

>      - `Installer (.exe) <https://repository.microej.com/packages/SDK/22.06/MicroEJ-SDK-Installer-Win64-22.06.exe>`__

Author: We can also duplicate all lines with the Portable package link
```suggestion
     - -`Installer (.exe) <https://repository.microej.com/packages/SDK/22.06/MicroEJ-SDK-Installer-Win64-22.06.exe>`__
       -`Portable (.zip) <https://repository.microej.com/packages/SDK/22.06/zip/microej-sdk-22.06-linux_x86_64.zip>`__ 
```

Author: @denisbedin could you please upload the missing portable files :
- https://repository.microej.com/packages/SDK/20.10/
- https://repository.microej.com/packages/SDK/20.07/
- https://repository.microej.com/packages/SDK/19.02/
- 
Thanks

     - `N/A`
     - ``5.4.0``
     - ``2020-06`` 
   * - ``20.12``
     - ``8``
     - `Installer (.exe) <https://repository.microej.com/packages/SDK/20.12/MicroEJ-SDK-Installer-Win64-20.12.exe>`__
     - `Installer (.zip) <https://repository.microej.com/packages/SDK/20.12/MicroEJ-SDK-Installer-Linux64-20.12.zip>`__
     - `Installer (.zip) <https://repository.microej.com/packages/SDK/20.12/MicroEJ-SDK-Installer-MacOS-20.12.zip>`__
     - `N/A`
     - ``5.3.1``
     - ``2020-06`` 
   * - ``20.10``
     - ``8``
     - `Installer (.exe) <https://repository.microej.com/packages/SDK/20.10/MicroEJ-SDK-Installer-Win64-20.10.exe>`__
     - `Installer (.zip) <https://repository.microej.com/packages/SDK/20.10/MicroEJ-SDK-Installer-Linux64-20.10.zip>`__
     - `Installer (.zip) <https://repository.microej.com/packages/SDK/20.10/MicroEJ-SDK-Installer-MacOS-20.10.zip>`__
     - `N/A`
     - ``5.3.0``
     - ``2020-06`` 
   * - ``20.07``
     - ``8``
     - `Installer (.exe) <https://repository.microej.com/packages/SDK/20.07/MicroEJ-SDK-Installer-Win64-20.07.exe>`__
     - `Installer (.zip) <https://repository.microej.com/packages/SDK/20.07/MicroEJ-SDK-Installer-Linux64-20.07.zip>`__

>      - `Installer (.zip) <https://repository.microej.com/packages/SDK/20.07/MicroEJ-SDK-Installer-MacOS-20.07.zip>`__

Reviewer 2: @cblanchard-microej I have uploaded the missing installers so you should not have the 404 error anymore

>      - `Installer (.zip) <https://repository.microej.com/packages/SDK/20.07/MicroEJ-SDK-Installer-MacOS-20.07.zip>`__

Reviewer 3: Yes, it works for me now.

     - `N/A`
     - ``5.2.0``
     - ``4.7.2`` 
   * - ``19.05``
     - ``8``
     - `Installer (.exe) <https://repository.microej.com/packages/SDK/19.05/MicroEJ-SDK-Installer-Win64-19.05.exe>`__
     - `Installer (.zip) <https://repository.microej.com/packages/SDK/19.05/MicroEJ-SDK-Installer-Linux64-19.05.zip>`__

>      - `Installer (.zip) <https://repository.microej.com/packages/SDK/19.05/MicroEJ-SDK-Installer-MacOS-19.05.zip>`__

Reviewer 3: I got a 404 Not Found issue with this links. I just find the Windows installer in this repository: https://repository.microej.com/packages/SDK/19.05/
